spec/helper.rb

```diff
@@ -28,9 +28,7 @@
   config.before(:all) do
     @app_key = '3ovRYNIb6OIAI0ddS7EfpYkHpi0oTKl6bj82Vjiy'
     @secret = 'ZeAbRfZQl8dIohaVuPiFPLbc9a0P1wdFRuOuk6ON'
-    VCR.use_cassette('get_oauth_token') do
-      @utoken ||= Yotpo.get_oauth_token({ app_key: @app_key, secret: @secret }).body.access_token
-    end
+    @utoken ||= 'Ufo5XkKXNsuh0k9ZMPndRv4DlV9324ptg730KKqC'
```

Comment:
I pulled the value out of the capture since this is just set up code and we're not interested in checking the validity of the `Yotpo.get_oauth_token` request & response

https://github.com/bespokepost/yotpo-ruby/blob/b8d749d0c9b34b546938eea0d0be0834a26ac952/spec/cassettes/get_oauth_token.yml#L50